app/docs/0.13.x/admin-api.md

```diff
@@ -1885,8 +1824,8 @@ HTTP 204 No Content
 
 The upstream object represents a virtual hostname and can be used to loadbalance
 incoming requests over multiple services (targets). So for example an upstream
-named `service.v1.xyz` with an API object created with an `upstream_url=https://service.v1.xyz/some/path`.
-Requests for this API would be proxied to the targets defined within the upstream.
+named `service.v1.xyz` with an Service object created with an `url=https://service.v1.xyz/some/path`.
```

Comment:
"a Service"

Reply:
We also need to refer to a Service's `host` here (and not `url`) since it is the only field that matters for linking the Upstream with a Service.